src/Features/VisualBasic/Portable/Completion/CompletionProviders/PartialTypeCompletionProvider.vb

```diff
@@ -109,15 +110,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
 
         Private Function CreateCompletionItem(symbol As INamedTypeSymbol,
                                               textSpan As TextSpan,
-                                              displayService As ISymbolDisplayService,
                                               position As Integer,
                                               context As VisualBasicSyntaxContext) As CompletionItem
             Dim displayText As String = Nothing
             Dim insertionText As String = Nothing
 
             If symbol.MatchesKind(SymbolKind.NamedType) AndAlso symbol.GetArity() > 0 Then
```

Comment:
This seems redundant. Can an `INamedTypeSymbol` ever _not_ be of kind `NamedType`?